Ductus.FluentDocker.Tests/FluentApiTests/FluentContainerBasicTests.cs

```diff
@@ -175,7 +175,7 @@ public void ExplicitPortMappingShouldWork()
 
     [TestMethod]
     [TestCategory("CI")]
-    public void ImplicitPortMappingShouldWork()
```

Comment:
Thanks for the PR! :) -  Can you please rename this function back to it's original name, unless there's a more implicit way doing ExporePort() that we need to have a unit test for?

Reply:
I just force-pushed 3567396b55bd1264059a54542e1cf58894bcf793 which does not alter the name of that test.